examples/node-app-mssql/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  api:
+    container_name: rest-api
+    image: rest-api
+    restart: always
+    build: .
+    ports:
+      - "3000:3000"
+    links:
+      - db
+  db:
+    image: mysql:5.7
```

Comment:
The example uses mssql so you'll need to use the official MS SQL docker image instead.

Reply:
Uses this image `mcr.microsoft.com/mssql/server:2017-GA-ubuntu`. Take this as a reference https://github.com/laudio/pyodbc/blob/master/examples/app-mssql/docker-compose.yml